app/views/hyrax/homepage/_featured_collection_fields.html.erb

```diff
@@ -4,7 +4,7 @@
   </div>
     <div class= 'homepage-work-thumbnail'>
       <%= link_to [hyrax, featured] do %>
-          <%= render_thumbnail_tag(featured, {suppress_link: true}) %>
+          <%= render_thumbnail_tag(featured.solr_document, {suppress_link: true}) %>
```

Comment:
Reading note: Ensuring that we're sending the SOLR document; because sometimes we have a SOLR doc and sometimes we have a presenter that wraps a SOLR doc.